doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,86 +1,123 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 Perception Pipeline Tutorial
-============================
+==================================
 
+1 Introduction
```

Comment:
Remove the numbers before the headings to match the style of the other tutorials
```suggestion
Introduction
```